.github/workflows/benchmark-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Run the benchmark tests
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: false
+        type: string
+        default: Cpp
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1.4.3
+        with:
+          java-version: 14
```

Comment:
Do we use Java 11 or 14? This inconsistency already existed in `master`, but I wonder why we have it.

Reply:
I wondered about this, too. Maybe @cmnrd can comment on this?

Reply:
Changed it to 11, and it appears to work fine.